docs/source/conf.py

```diff
@@ -30,7 +30,9 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    "sphinx.ext.autodoc",
+    # Generate complete API docs by parsing source code
+    "autoapi.extension",
+    # Add links to source from generated docs
```

Comment:
Thanks for adding these comments! It will make it much easier to reuse this template elsewhere